### ISSUE TITLE
DPL-444 - Create automated 10X and manual 10X submissions from the same cherrypick plate

### ISSUE THREAD

**Issue by stevieing** [sanger/limber#1131]
**User story**
 SSRs need the ability to create both ‘auto 10X’ and ‘manual 10X’ submissions from the same cherry pick plate.  Multiple 10X auto submissions are possible (for example a 5’GEX and 5’TCR’) but SSRs have said it’s not possible to make a 10X auto submission along with a manual submission (so for example a 10X auto 5’GEX plus a manual 5’ TCR) from the **same** source plate (ie. either an LBB Cherrypick or an LBC Cherrypick).

**Who are the primary contacts for this story**
Matt / Siobhan?

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Ability to create both ‘auto 10X’ and ‘manual 10X’ submissions from either the LBB or LBC Cherrypick plate

**Dependencies**
This story is blocked by the following dependencies:
- #1129

**Notes**
Change the pipeline configs for bespoke_pcr and all the bespoke_chromium pipelines to allow either LBB or LBC Cherrypicks to be the starting point.

**Additional context**
For background, the automated / high throughput Chromium 10 pipelines in Limber are:
- Bespoke Chromium 3pv3
- Bespoke Chromium 5p
- Bespoke Chromium BCR
- Bespoke Chromium TCR

And the manual pipeline is:
- Bespoke PCR

Bespoke Chromium 3pv2 was also manual but not used any more.

**Comment by stevieing** [comment]
don't understand what they are asking for there. Sounds like new submission templates which is just config, but don't follow what they're saying about combinations of manual and auto submissions.

**Comment by andrewsparkes** [comment]
Submission templates using the new library types should allow this.
Consider one submission for both manual and auto, what about billing if use the same request type / library type.
If same then also would get 5 or 6 suggested actions on the LBC Cherrypick plate in Limber.
Seems likely they need 2 submissions but can test.

**Comment by andrewsparkes** [comment]
James strongly recommending to use a different library type for bespoke vs automated routes.
Billing depends on library type and request type, but not plate purpose, and only library type is exposed in MLWH.
Submissions cannot mix request types, and limber suggested actions use them to determine suggested actions, so best to do a distinct submission for bespoke vs automated, and use different library types.

**Comment by andrewsparkes** [comment]
Conclusion is we cannot mix two submission templates (request types) so this story is not possible.

**Comment by andrewsparkes** [comment]
Want to allow 'LBC Cherrypick' to be an option as the first plate purpose in both the bespoke_pcr (where they currently start with 'LBB Cherrypick') and the automated bespoke_chromium pipelines (which start with 'LBC Cherrypick'). Should be just an addition to the config for bespoke_pcr but will need testing. 
NB. LBC Cherrypick is also the output plate from the bespoke_aggregation pipeline, but this pipeline may not be being used any more (so could check with them and create story to remove it if no longer needed).

**Comment by KatyTaylor** [comment]
Deployment script:
```
p = Purpose.find_by(name: 'LBC Cherrypick')
p.type
p.update!(type: 'PlatePurpose::Input')

p.cherrypickable_target
p.update!(cherrypickable_target: true)
```

Changes LBC Cherrypick to be:
a) An input plate - this means that Limber allows it to be used as the first plate in the pipeline, even if it hasn't got transfer requests into it or not. I'm not quite sure if this is a realistic scenario - it came up during UAT but could be because we were cutting corners (not doing the aggregation pipeline) to get to the main part of the testing. I don't think it causes harm though.
b) Able to be cherrypicked into - this came up during UAT - it could help as the team mostly use the aggregation pipeline to do a straightforward cherrypick - and the intermediate plate in that is a 'dummy' plate - just in the LIMS and not in reality. Cherrypicking could be an alternative, quicker way to get what they need to start the Limber pipeline.
